pkg/networkmanager/interfaces.js

```diff
@@ -1006,7 +1006,7 @@ export function NetworkManagerModel() {
                                               settings_to_nm(settings), objpath(this), objpath(specific_object))
                             .then(([path, active_connection]) => active_connection);
                 } catch (e) {
-                    return cockpit.reject(e);
+                    return Promise.reject(e);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18487-20230310-143650-f841503b-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)